Assets/Scripts/OverlayManager.cs

```diff
@@ -628,6 +637,14 @@ private IEnumerator<Null> ConsumeAsProgress(IEnumerator<float> coroutine)
 
         public void RenderLogo(double progress)
         {
+            // TODO:Mike Temp hack to set correct logo progress
```

Comment:
Do these 'temp' hacks need to be fixed for this to land, or will they be acceptable as is?

Reply:
Yeah this will do for now, this was temp until unity has proper compositor support again!